.github/workflows/release.yaml

```diff
@@ -364,5 +373,6 @@ jobs:
             dist/argo-*.gz
             dist/argo-*.gz.sha256
             dist/manifests/*.yaml
+            dist/*.spdx
```

Comment:
Those spdx files will be large text files. In ArgoCD and ArgoRollouts we are going to provide all spdx files packed and compacted with `tar.gz` in a file called `sbom.tar.gz`. Maybe we should maintain consistency across the projects and do the same for ArgoWorkflows?

Reply:
I don't think I'm worried about size, but useful to download them all in a single file. Will update